ports/riot/summit.mk

```diff
@@ -0,0 +1,15 @@
+all: build
+
+GENHDR_DIR := ../../genhdr
+
+RIOT:
+	git -c advice.detachedHead=false \
+		clone --branch 2021.04 https://github.com/RIOT-OS/RIOT
```

Comment:
FIXME: when using RIOT of latest master, build fails with `collect2: error: ld returned 1 exit status`